pkg/config/deployment.go

```diff
@@ -29,6 +29,10 @@ const (
 )
 
 type GenericDeploymentSpec struct {
+	Name   string            `json:"name"`
+	EnvID  string            `json:"env_id"`
```

Comment:
```suggestion
	EnvID  string            `json:"envId"`
```

Reply:
Btw, isn't the name better than the id?

Reply:
I was worried about the uniqueness of env_name but let's leave it to the user to maintain the uniqueness.

It's obvious that the application name shouldn't be unique, but ideally the env name should be unique because Environment is a project-wide concept.
To do that with a document-oriented database, we need to prepare some kind of extra table to ensure uniqueness.

But we're planning to migrate Env to Labels in the future, again, for now let's leave it to the user to maintain the uniqueness.

Reply:
If we proceed with the Name as non-unique, we will have to add a not neat method to `environmentstore`. I'd like to hear your opinion on uniqueness before implementing it.